front/src/modules/conflict/components/ConflictsList.tsx

```diff
@@ -13,20 +14,132 @@ type ConflictsListProps = {
   conflicts: Conflict[];
   timetableItems: TimetableItemWithDetails[];
   onConflictClick: (conflict: ConflictWithTrainNames) => void;
+  showOnlySelectedTrain?: boolean;
 };
 
-const ConflictsList = ({ conflicts, timetableItems, onConflictClick }: ConflictsListProps) => {
+const ConflictsList = ({
+  conflicts,
+  timetableItems,
+  onConflictClick,
+  showOnlySelectedTrain = false,
+}: ConflictsListProps) => {
+  const selectedTrainId = useSelector(getSelectedTrainId);
+
   const enrichedConflicts = useMemo(
     () => addTrainNamesToConflicts(conflicts, timetableItems),
     [conflicts, timetableItems]
   );
+
+  const selectedTrainName = useMemo(() => {
+    if (!selectedTrainId) return null;
+
+    let selectedTrain = timetableItems.find((item) => item.id === selectedTrainId);
+
+    if (!selectedTrain && selectedTrainId.startsWith('indexedoccurrence_')) {
+      const pacedTrainId = `paced_${selectedTrainId.split('_')[1]}`;
+      selectedTrain = timetableItems.find((item) => item.id === pacedTrainId);
+    }
```

Comment:
We can use existings [utils](https://github.com/OpenRailAssociation/osrd/blob/dev/front/src/utils/trainId.ts) function
```suggestion
    if (!selectedTrain && isIndexedOccurrenceId(selectedTrainId)) {
      const pacedTrainId = extractPacedTrainIdFromOccurrenceId(selectedTrainId);
      selectedTrain = timetableItems.find((item) => item.id === pacedTrainId);
    }
```

Reply:
Done